media/src/components/chat/Sidebar.tsx

```diff
@@ -0,0 +1,26 @@
+import { MdSettings, MdDashboard } from 'react-icons/md'
+import { HiBell } from 'react-icons/hi'
+import { IconButton, Tooltip, VStack } from '@chakra-ui/react'
+
+
```

Comment:
extra spaces? maybe consider adding a .prettier config and have your IDE format on save?